### PR TITLE
Ng16

### PR DESCRIPTION
**Pull request by stevermeister**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,29 +12,29 @@
...
-    "@angular-eslint/template-parser": "^15.2.0",
+    "@angular-devkit/build-angular": "^16.0.0",
+    "@angular-eslint/builder": "^16.0.1",
+    "@angular-eslint/eslint-plugin": "^16.0.1",
+    "@angular-eslint/eslint-plugin-template": "^16.0.1",
+    "@angular-eslint/schematics": "^16.0.1",
+    "@angular-eslint/template-parser": "^16.0.1",
+    "@angular/cli": "^16.0.0",
+    "@angular/compiler-cli": "^16.0.0",
+    "@angular/language-service": "^16.0.0",
```
Comment (bampakoa) [inline package.json:37]: Angular Language Service is not needed to be present in the `package.json` file anymore.

File: projects/ngx-wig/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-wig",
-  "version": "15.1.3",
+  "version": "15.1.4",
```
Comment (bampakoa) [inline projects/ngx-wig/package.json:3]: Shouldn't we set it as 16.0.0 instead?

Comment (stevermeister) [reply projects/ngx-wig/package.json:3]: yes, indeed!

**Comment by stevermeister** [comment]
@bampakoa updated

**Review by bampakoa — APPROVED** [review]
Looks great 👍

**Pull request merged**